Tests/AfluentTests/TestHelpers/TestError.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  TestError.swift
+//  Afluent
+//
+//  Created by Roman Temchenko on 2025-03-07.
+//
+
+import Foundation
+
+enum TestError: Error, Equatable {
```

Comment:
This particular error might not make a great candidate for extraction. It's kind of test specific, sometimes it needs to be equatable, sometimes it might need to conform to other protocols, or have other cases, or associated types, etc.